src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Utility/PresidioUtility.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Anonymizer.Core.Utility
+{
+    public class PresidioUtility
+    {
+        public static string Anonymize(string text)
```

Comment:
I am not sure it's the best practice to create a new instance of the SDK on each use.
Perhaps, static is not the best fit?

Reply:
Agree, wanted to be aligned with their approach at first. Processor -> calling static Utility method.
We might leave it static and instantiate PresidioUtility with the client arguments for the SDK. 
Currently we don't need to have this decision though, in my opinion. YAGNI
@navalev WDYT?

Reply:
The presidio utility can create a single instance of the SDK, but it can be done later in a seperate PR.

Reply:
The utility can be injected easily and let the DI handle the life span of the object. I don't think we should merge it like that. Assuming you have an SDK, how will you handle it?